src/modules/AuthModule/components/ForgetPass/ForgetPass.tsx

```diff
@@ -15,10 +15,12 @@ import axios from "axios";
 import forgotImage from "../../../../assets/Group 33.png";
 import IconDarkLightMode from "../../../../modules/SharedModule/Mode/IconDarkLightMode";
 import { toast } from "react-toastify";
+import { AuthContext } from "../../../../Context/AuthContext";
 
 export default function ForgetPass() {
-  // const { baseUrl} = useContext(AuthContext);
-  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const { baseUrl } = useContext(AuthContext);
+
+  const [isSubmitting, setisSubmitting] = useState<boolean>(false);

```

Comment:
No need for the state, we can get it from the useForm hook directly from formState
```
 formState: { errors , isSubmitting}
```